frontend-instructor/src/components/navBar/NavItems.tsx

```
@@ -0,0 +1,43 @@
+
+          <Link
+            to="/my-courses"
+            className="px-4 py-2 rounded-md text-sm font-medium transition-all bg-gray-800 text-white hover:bg-gray-700"
+          >
+            My Courses
+          </Link>
+
+          <button
+            onClick={handleLogOut}
+            className="px-4 py-2 rounded-md text-sm font-medium transition-all bg-red-600 text-white hover:bg-red-500"
+          >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)